src/screens/ProfileScreen/elements/ProfilePicture.js

```diff
@@ -1,61 +1,69 @@
 import * as React from 'react';
-import { StyleSheet, TouchableNativeFeedback, View } from 'react-native';
+import {StyleSheet, TouchableNativeFeedback, View} from 'react-native';
 
+import FastImage from 'react-native-fast-image';
+import PropTypes from 'prop-types';
 import Image from '../../../components/Image';
 import MemoIcAddCircle from '../../../assets/icons/ic_add_circle';
-import { DEFAULT_PROFILE_PIC_PATH } from '../../../utils/constants';
-import FastImage from 'react-native-fast-image';
+import {DEFAULT_PROFILE_PIC_PATH} from '../../../utils/constants';
+import {normalize} from '../../../utils/fonts';
+import {CircleGradient} from '../../../components/Karma/CircleGradient';
 
-/**
- * @typedef {Object} ProfilePicturePropsParam
- * @property {function} onImageContainerClick
- * @property {String} profilePicPath
- * @property {Boolean} disabledAddIcon
- */
-/**
- *
- * @param {ProfilePicturePropsParam} props
- */
-const ProfilePicture = ({ onImageContainerClick, profilePicPath, disabledAddIcon = false}) => {
-    const __renderAddIcon = () => {
-        if(!profilePicPath || !disabledAddIcon) return <></>
+const ProfilePicture = ({
+  onImageContainerClick,
+  profilePicPath,
+  disabledAddIcon = false,
+  karmaScore = 0
+}) => {
+  const renderAddIcon = () => {
+    if (!profilePicPath || !disabledAddIcon) return <></>;
 
-        return <MemoIcAddCircle width={48} height={48} style={styles.addCircle} />
-    }
-    return <View style={styles.wrapImageProfile}>
-        <TouchableNativeFeedback onPress={onImageContainerClick}>
-            <View style={styles.profileImageContainer}>
+    return <MemoIcAddCircle width={48} height={48} style={styles.addCircle} />;
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition in the `renderAddIcon` function seems to be incorrect. The add icon should be rendered when there is no profile picture and the `disabledAddIcon` prop is false. However, the current condition will not render the add icon if either of these conditions is true. Consider changing the logical OR operator (`||`) to a logical AND operator (`&&`).
```diff
-    if (!profilePicPath || !disabledAddIcon) return <></>;
+    if (!profilePicPath && !disabledAddIcon) return <></>;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->